Dockerfile

```diff
@@ -7,12 +7,20 @@ RUN mkdir -p $GOBIN
 
 COPY . $GOPATH/src/github.com/grafana/grafana
```

Comment:
Pretty minor, but in the other images I've been setting the WORKDIR, and that would simplify later cd/cp commands.
```
WORKDIR /go/src/github.com/prometheus/prometheus
COPY . .
```